tools/otsdb-dump-restore/build-tools/setup-devel-user.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+set -eux
+
+echo "$@"
+
+DEVEL_NAME="${1}"
+DEVEL_UID="${2}"
+DEVEL_GID="${3}"
```

Comment:
Maybe some checking for parameters and some help message here saying to execute it with sudo?

Reply:
This is not supposed to be used by the user directly. The user must use the Makefile, but I think I should explain this better.

Reply:
There is no test for it. I thought it would better for the script to fail explicitly rather than continue for example with an empty UID or GID.

Reply:
I was adding some lines to check if the variables are empty. When I tested, it does fail before checking:
<img width="913" alt="Screenshot 2023-03-22 at 16 47 48" src="https://user-images.githubusercontent.com/13034720/226962571-73dd2fc4-cf28-4ebd-9049-1ceec5304937.png">
I'd say it's enough with this error. If not, I can add some ifs before the argument assignment but maybe it is redundant. What do you think?

Reply:
aah, that's the `set -u` does the magic: 
```
-u  Treat unset variables as an error when substituting.
```

all is good then